### PR TITLE
Updates/v3.8

### PR DESCRIPTION
**Pull request by Houzvicka**
As discussed in the #30 
Update of library to version 3.8

### CODE REVIEW

File: Src/Xceed.Wpf.Toolkit/MessageBox/Implementation/MessageBox.cs [hunk 1]
```diff
@@ -932,7 +932,7 @@ private void SetImageSource( MessageBoxImage image )
       }
 
       // Use this syntax for other themes to get the icons
-      this.ImageSource = new BitmapImage( new Uri( String.Format( "/DotNetProjects.Wpf.Extended.Toolkit;component/MessageBox/Icons/{0}", iconName ), UriKind.RelativeOrAbsolute ) );
+      this.ImageSource = new BitmapImage( new Uri( String.Format( "/Xceed.Wpf.Toolkit;component/MessageBox/Icons/{0}", iconName ), UriKind.RelativeOrAbsolute ) );
```
Comment (jogibear9988) [inline Src/Xceed.Wpf.Toolkit/MessageBox/Implementation/MessageBox.cs]: Will this work? As the Project is not named Xceed

File: Src/Xceed.Wpf.Toolkit/Themes/Aero.NormalColor.xaml [hunk 2]
```diff
@@ -19,8 +19,8 @@
                     xmlns:core="clr-namespace:Xceed.Wpf.Toolkit.Core">
 
    <ResourceDictionary.MergedDictionaries>
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Themes/Aero/Brushes_NormalColor.xaml"/>
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit"
+      <core:VersionResourceDictionary AssemblyName="Xceed.Wpf.Toolkit" SourcePath="Themes/Aero/Brushes_NormalColor.xaml"/>
```
Comment (jogibear9988) [inline Src/Xceed.Wpf.Toolkit/Themes/Aero.NormalColor.xaml]: Same here...

File: Src/Xceed.Wpf.Toolkit/Themes/Aero2.NormalColor.xaml [hunk 3]
```diff
@@ -19,46 +19,46 @@
...
-
-
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/TimePicker/Themes/Aero2.NormalColor.xaml" />
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/TimeSpanUpDown/Themes/Aero2.NormalColor.xaml" />
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/WatermarkComboBox/Themes/Aero2.NormalColor.xaml" />
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/WatermarkTextBox/Themes/Aero2.NormalColor.xaml" />
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/Wizard/Themes/Aero2.NormalColor.xaml" />
-        <ResourceDictionary Source="/DotNetProjects.Wpf.Extended.Toolkit;component/Zoombox/Themes/Aero2.NormalColor.xaml" />
-    </ResourceDictionary.MergedDictionaries>
+      <ResourceDictionary Source="/Xceed.Wpf.Toolkit;component/Themes/Aero2/Brushes.xaml" />
```
Comment (jogibear9988) [inline Src/Xceed.Wpf.Toolkit/Themes/Aero2.NormalColor.xaml]: And here

File: Src/Xceed.Wpf.Toolkit/Themes/Generic.xaml [hunk 4]
```diff
@@ -18,50 +18,45 @@
...
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TimePicker/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TimeSpanUpDown/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="WatermarkComboBox/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="WatermarkTextBox/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Wizard/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Zoombox/Themes/Generic.xaml" />
-
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TokenizedTextBox/Themes/Generic.xaml" />
-    </ResourceDictionary.MergedDictionaries>
+      <core:VersionResourceDictionary AssemblyName="Xceed.Wpf.Toolkit" SourcePath="Themes/Generic/Brushes.xaml" />
```
Comment (jogibear9988) [inline Src/Xceed.Wpf.Toolkit/Themes/Generic.xaml]: and here

Comment (Houzvicka) [reply Src/Xceed.Wpf.Toolkit/Themes/Generic.xaml]: I overlooked these. And also create a simple app to test, and you are right. So I made the changes, and now it works correctly :)

**Comment by jogibear9988** [comment]
lgtm

**Comment by jogibear9988** [comment]
thanks

**Pull request merged**